tests/v1/entrypoints/llm/test_struct_output_generate.py

```diff
@@ -403,3 +320,41 @@ def test_guided_json_completion_with_enum(
         print(f"Prompt: {prompt!r}, Generated text: {generated_text!r}")
         output_json = json.loads(generated_text)
         jsonschema.validate(instance=output_json, schema=json_schema)
+
+
+@pytest.mark.skip_global_cleanup
+@pytest.mark.parametrize("model_name", MODELS_TO_TEST)
+def test_structured_output_auto_mode(
+    monkeypatch: pytest.MonkeyPatch,
+    unsupported_json_schema: dict[str, Any],
+    model_name: str,
+):
+    monkeypatch.setenv("VLLM_USE_V1", "1")
+
+    llm = LLM(model=model_name,
```

Comment:
seeing `enforce_eager=True` will further reduce the time by about 60s

Reply:
Many of the tests have `max_tokens=1000` and are generation focused, so this may be an unclear tradeoff

Reply:
addiing it shaved off 4 more minutes -- down to about 2.5 minutes now